examples/ordering/src/main.rs

```diff
@@ -27,6 +26,23 @@ struct Args {
     n_items: usize,
 }
 
+fn create_backup(node_id: usize) -> Result<(File, io::Cursor<Vec<u8>>), io::Error> {
+    let stash_path = Path::new("./aleph-bft-examples-ordering-backup");
+    fs::create_dir_all(&stash_path)?;
+    let file_path = stash_path.join(format!("{}.units", node_id));
+    let _ = fs::OpenOptions::new()
+        .create_new(true)
+        .write(true)
+        .append(true)
+        .open(file_path.clone());
```

Comment:
This is a little confusing -- this will create the file in case it does not exist, correct? Would
```
    let _ = fs::OpenOptions::new().create_new(true).open(file_path.clone());
```
not be enough?

Reply:
both `create` and `create_new` require `write` or `append` to work.

Reply:
replaced with `let _ = fs::OpenOptions::new().create(true).append(true).open(file_path.clone());`